client/web/src/user/settings/auth/UserSettingsSecurityPage.tsx

```diff
@@ -92,8 +92,11 @@ export const UserSettingsSecurityPage: React.FunctionComponent<React.PropsWithCh
     }
 
     // auth providers by service ID
-    const accountByServiceID = accounts.fetched?.reduce((accumulator: AccountByServiceID, account) => {
-        accumulator[account.serviceID] = account
+    const accountsByServiceID = accounts.fetched?.reduce((accumulator: AccountsByServiceID, account) => {
+        let accountMap = accumulator[account.serviceID] ?? []
+        accountMap.push(account)
+        accumulator[account.serviceID] = accountMap
```

Comment:
Nice try, Mr. Array.

```suggestion
        let accountArray = accumulator[account.serviceID] ?? []
        accountArray.push(account)
        accumulator[account.serviceID] = accountArray
```

Reply:
And I would have gotten away with it to, if it weren't for that meddling kid! - Mr. Array, probably